intel_extension_for_pytorch/_inductor/xpu/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .overrides import override_decode_device
+from .utils import has_triton
+
+
+if has_triton():
+    override_decode_device()
```

Comment:
It's better to add some comments here to explain why we must override `decode_device`.